src/Npgsql/NpgsqlTypes/NpgsqlTypes.cs

```diff
@@ -200,7 +200,7 @@ void NormalizeBox()
 /// </summary>
 public struct NpgsqlPath : IList<NpgsqlPoint>, IEquatable<NpgsqlPath>
 {
-    readonly List<NpgsqlPoint> _points;
+    List<NpgsqlPoint> _points;
```

Comment:
By the way, why a list and not an array? Is it just the way it is or is there a reason?